### PR TITLE
dev: add option to use postgresql in dev environment

### PR DESCRIPTION
**Pull request by phillimorland**

### CODE REVIEW

**Comment by phillimorland** [comment]
@goapunk @fuzzylogic2000 sorry guys, was on auto pilor this morning and was on wrong pull request when merged, have reverted but it when restored this didn't open so just opened a new one, here is link https://github.com/liqd/a4-meinberlin/pull/3817

**Comment by goapunk** [comment]
@fuzzylogic2000 ping me if you want to test this again :)

**Comment by goapunk** [comment]
I updated this so it's using a different port for each project so it should work with multiple instances now

File: README.md [hunk 1]
```diff
@@ -22,3 +22,15 @@ based on [adhocracy 4](https://github.com/liqd/adhocracy4).
...
+### Use postgresql database for testing:
+run the following command once:
+```
+make create-postgres
+```
+to start the testserver with postgresql, run:
+```
+export DATABASE=postgresql
+make start-postgres
+make watch
```
Comment (sabinammm) [inline README.md:35]: I had to also do 
`python manage.py migrate`
Without it, I get `ProgrammingError: relation "django_session" does not exist`. 
Maybe add this here?

Comment (goapunk) [reply README.md:35]: that should be solved if you do `make install`, that shouldn't have to go here

Comment (sabinammm) [reply README.md:35]: Oh, sorry, my mistake - I did it in the order in the readme.

**Review by fuzzylogic2000 — APPROVED** [review]
So cool!

**Pull request merged**